src/TestLogger/Extensions/NUnitTestAdapter.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Spekt Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Spekt.TestLogger.Extensions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using Spekt.TestLogger.Core;
+
+    public class NUnitTestAdapter : ITestAdapter
+    {
+        private const string ExplicitLabel = "Explicit";
+
+        public List<TestResultInfo> TransformResults(List<TestResultInfo> results, List<TestMessageInfo> messages)
+        {
+            foreach (var result in results)
+            {
+                // Mark tests with Explicit attribute as Skipped instead of Inconclusive.

```

Comment:
I think this could be a bit more detailed about what is going on. First that the Explicit attribute in nunit gets passed as a trait to us. The nunit docs https://docs.nunit.org/articles/nunit/writing-tests/attributes/explicit.html made it clear why explicit lead to skipped, which wasn't obvious to me either.